.github/workflows/manual-deploy.yml

```diff
@@ -0,0 +1,42 @@
+name: Manual Deploy
+concurrency: build_and_deploy_v2_${{ github.ref_name }}
```

Comment:
This doesn't fit existing. Current has https://github.com/DFE-Digital/access-your-teaching-qualifications/blob/main/.github/workflows/build-and-deploy.yml#L109 so move this to deploy step here. Might need to add concurrency here too https://github.com/DFE-Digital/access-your-teaching-qualifications/blob/main/.github/workflows/build-and-deploy.yml#L137